src/vm/functions/sequences.rs

```diff
@@ -311,3 +311,30 @@ pub fn native_element_at(sequence: Value, index: Value) -> Result<Value> {
         Ok(Value::none())
     }
 }
+
+pub fn special_slice(
+    args: &[SymbolicExpression],
+    env: &mut Environment,
+    context: &LocalContext,
+) -> Result<Value> {
+    check_argument_count(3, args)?;
+
+    // TODO: runtime_cost
+
+    let seq = eval(&args[0], env, context)?;
+    let position = eval(&args[1], env, context)?;
+    let length = eval(&args[2], env, context)?;
+
+    let sliced_seq = match (seq, position, length) {
+        (Value::Sequence(seq), Value::UInt(position), Value::UInt(length)) => {
+            let (position, length) = match (usize::try_from(position), usize::try_from(length)) {
```

Comment:
`usize` has a maximum value that is dependent on the host architecture -- it's not guaranteed to have a given number of bits.  Can you use something else, like `u32` or `u128` or something?